src/ebonite/ext/ext_loader.py

```diff
@@ -60,7 +60,8 @@ class ExtensionLoader:
         Extension('ebonite.ext.flask', ['flask', 'flaskgger']),
         Extension('ebonite.ext.sqlalchemy', ['sqlalchemy']),
         Extension('ebonite.ext.s3', ['boto3']),
-        Extension('ebonite.ext.imageio', ['imageio'])
+        Extension('ebonite.ext.imageio', ['imageio']),
+        Extension('ebonite.ext.xgboost', ['xgboost'])
```

Comment:
I don't thnk that XGBoost should be mandatory extension

Reply:
done